timelock-agent/src/test/java/com/palantir/timelock/config/ClusterConfigurationTest.java

```diff
@@ -42,11 +43,14 @@ public void shouldThrowIfLocalServerNotInServers() {
         assertThatIllegalArgumentException().isThrownBy(builder::build);
     }
 
+    // Todo(snanda): Remove ignore when ClusterInstallConfig is killed
```

Comment:
nit: I think these are normally `TODO` (all caps).

Also, would prefer if we explain that the check happens at a different level (to not give the impression that this is ignored because we are allowing such configurations without the disclaimer).